web/massastation/src/pages/Index/Index.tsx

```diff
@@ -123,22 +127,37 @@ export function Index() {
                 <Image4Light />,
                 <Image5Light />,
               ]}
-              components={[
-                <PluginWallet
-                  key="wallet"
-                  isActive={pluginWalletIsInstalled}
-                  title="Massa Wallet"
-                  iconActive={<WalletActive />}
-                  iconInactive={<WalletInactive />}
-                  onClickActive={() =>
-                    window.open(
-                      '/plugin/massa-labs/massa-wallet/web-app/index',
-                      '_blank',
-                    )
-                  }
-                  onClickInactive={handleInstallPlugin}
-                />,
-              ]}
+              components={
+                !isLoading
+                  ? [
+                      <PluginWallet
+                        key="wallet"
+                        isActive={pluginWalletIsInstalled}
+                        title="Massa Wallet"
+                        iconActive={<WalletActive />}
+                        iconInactive={<WalletInactive />}
+                        onClickActive={() =>
+                          window.open(
+                            '/plugin/massa-labs/massa-wallet/web-app/index',
+                            '_blank',
+                          )
+                        }
+                        onClickInactive={handleInstallPlugin}
+                      />,
+                    ]
+                  : [
+                      <div>
+                        <WalletInactive />
+                        <Button
+                          type="submit"
```

Comment:
The button is set to be always disabled, isn't it... I don't think you need a type submit!?

Reply:
done